src/structures/Guild.js

```diff
@@ -510,7 +510,8 @@ class Guild extends Base {
    * @readonly
    */
   get me() {
-    return this.members.get(this.client.user.id) || (this.client.options.partials.includes(PartialTypes.GUILD_MEMBER) ?
+    // eslint-disable-next-line max-len
+    return this.members.cache.get(this.client.user.id) || (this.client.options.partials.includes(PartialTypes.GUILD_MEMBER) ?
```

Comment:
Is there a good reason to disable the eslint rule here? Would much prefer to keep the rule intact.